install.sh

```diff
@@ -1080,95 +842,10 @@ VMM_EXEC_LIB="src/vmm_mad/remotes/lib/command.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/virtual_wire.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/distributed_firewall.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsxt_dfw.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsxv_dfw.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/logical_port.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsxt_logical_port.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsxv_logical_port.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsx_rule.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsxt_rule.rb \
-                    src/vmm_mad/remotes/lib/nsx_driver/nsxv_rule.rb"
-
-#-------------------------------------------------------------------------------
-# VMM SH Driver LXD scripts, to be installed under $REMOTES_LOCATION/vmm/lxd
-#-------------------------------------------------------------------------------
-VMM_EXEC_LXD_SCRIPTS="src/vmm_mad/remotes/lxd/cancel \
-                    src/vmm_mad/remotes/lxd/deploy \
-                    src/vmm_mad/remotes/lxd/migrate \
-                    src/vmm_mad/remotes/lxd/migrate_local \
-                    src/vmm_mad/remotes/lxd/restore \
-                    src/vmm_mad/remotes/lxd/reboot \
-                    src/vmm_mad/remotes/lxd/reset \
-                    src/vmm_mad/remotes/lxd/save \
-                    src/vmm_mad/remotes/lxd/attach_disk \
-                    src/vmm_mad/remotes/lxd/detach_disk \
-                    src/vmm_mad/remotes/lxd/attach_nic \
-                    src/vmm_mad/remotes/lxd/detach_nic \
-                    src/vmm_mad/remotes/lxd/snapshot_create \
-                    src/vmm_mad/remotes/lxd/snapshot_revert \
-                    src/vmm_mad/remotes/lxd/snapshot_delete \
-                    src/vmm_mad/remotes/lxd/shutdown \
-                    src/vmm_mad/remotes/lxd/reconfigure \
-                    src/vmm_mad/remotes/lxd/prereconfigure \
-                    src/vmm_mad/remotes/lxd/resize_disk"
-
-VMM_EXEC_LXD_LIB="src/vmm_mad/remotes/lib/lxd/opennebula_vm.rb \
-                src/vmm_mad/remotes/lib/lxd/mapper/mapper.rb \
-                src/vmm_mad/remotes/lib/lxd/mapper/qcow2.rb \
-                src/vmm_mad/remotes/lib/lxd/mapper/raw.rb \
-                src/vmm_mad/remotes/lib/lxd/mapper/rbd.rb \
-                src/vmm_mad/remotes/lib/lxd/client.rb \
-                src/vmm_mad/remotes/lib/lxd/command.rb \
-                src/vmm_mad/remotes/lib/lxd/container.rb"
-
 #-------------------------------------------------------------------------------
 # VMM SH Driver LXC scripts, to be installed under $REMOTES_LOCATION/vmm/lxc
 #-------------------------------------------------------------------------------
-VMM_EXEC_LXC_SCRIPTS="
-                    src/vmm_mad/remotes/lxc/attach_disk \
+VMM_EXEC_LXC_SCRIPTS="src/vmm_mad/remotes/lxc/attach_disk \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

VMM_EXEC_LXC_SCRIPTS appears unused. Verify use (or export if used externally).